content/tutorial/nav.yml

```diff
@@ -1,111 +1,111 @@
-- title: Tutorial
+- title: ट्यूटोरियल
   items:
     - id: before-we-start-the-tutorial
-      title: Before We Start the Tutorial
+      title: ट्यूटोरियल शुरू करने से पहले
       href: /tutorial/tutorial.html#before-we-start-the-tutorial
       forceInternal: true
       subitems:
         - id: what-are-we-building
-          title: What Are We Building?
+          title: हम क्या बना रहे हैं?
           href: /tutorial/tutorial.html#what-are-we-building
           forceInternal: true
         - id: prerequisites
-          title: Prerequisites
+          title: आवश्यक शर्तें
           href: /tutorial/tutorial.html#prerequisites
           forceInternal: true
     - id: setup-for-the-tutorial
-      title: Setup for the Tutorial
+      title: ट्यूटोरियल के लिए व्यवस्था
```

Comment:
व्यवस्था seems a bit odd here, should we just use the hindi word instead?

Reply:
@arshadkazmi42 still see `व्यवस्था` here, maybe you forgot to push local commits?